include/aws/common/byte_buf.h

```diff
@@ -562,6 +564,7 @@ AWS_STATIC_IMPL size_t aws_nospec_mask(size_t index, size_t bound) {
      * an arithmetic right shift operation on x86.
      */
     combined_mask = combined_mask * UINTPTR_MAX;
+#pragma CPROVER check pop
```

Comment:
I'd find it cleaner to put the pop on the last line of the function, after the return.  Alternatively, put the pragma around the while function, including the signature.